CHANGELOG.rst

```diff
@@ -17,6 +17,7 @@ v0.18.1
 - [ENH] add engineering submodule with unit conversion method by @rahosbach
 - [DOC] add PyPI project description
 - [ENH] add example notebook with use of finance submodule methods by @rahosbach
+- [DOC] fix example for ``update_where``, and provide a bit more details.
```

Comment:
This should be under the `release_number` (on deck) section.

Can you pull in latest changes from `dev` and push? That will show the latest changes, and possibly resolve the merge conflict that is happening.

Reply:
Sorry, silly mistake. Will fix.